tjhsst/templates/base.html

```diff
@@ -37,6 +37,10 @@ <h1>TJHSST</h1>
                             <button type="submit" class="search-button"><i class="fa fa-search"></i></button>
                         </form>
                     </li>
+                {% else %}
+                    <li>
+                        <span class="navbar-text" style="color:rgb(255,255,255)"><a href="{% url 'auth:login' %}">Login</a></span>
```

Comment:
1. I think you meant to add this to the `if` tag a few lines down.
2. From testing with this addition moved down there, this link feels out of place (and the default underline-on-hover behavior feels wrong). Perhaps further discussion is needed.
3. It turns out setting the color explicitly is unnecessary (it's set by Bootstrap). I know it's done in the username label a few lines down, but we should remove that too.